packages/angular_devkit/build_angular/src/webpack/plugins/common-js-usage-warn-plugin.ts

```diff
@@ -37,7 +37,10 @@ export class CommonJsUsageWarnPlugin {
   // Allow the below depedency for HMR
   // tslint:disable-next-line: max-line-length
   // https://github.com/angular/angular-cli/blob/1e258317b1f6ec1e957ee3559cc3b28ba602f3ba/packages/angular_devkit/build_angular/src/dev-server/index.ts#L605-L638
-  private allowedDepedencies = new Set<string>(['webpack/hot/dev-server']);
+  private allowedDepedencies = new Set<string>([
+    '@angular-devkit/build-angular',
```

Comment:
Is this needed?  The `hmr.js` has import statements so it should be considered ESM.

Reply:
Ah not anymore.